src/features/Miscellaneous/AdditionalInformation/index.tsx

```diff
@@ -26,89 +26,89 @@ const AdditionalInformation = ({
     | SystemIntakeFragmentFragment;
   type: RequestType;
 }) => {
-  const { t } = useTranslation('admin');
+  const { t: adminT } = useTranslation('admin');
+  const { t: linkedSystemsT } = useTranslation('linkedSystems');
 
   const parentRoute = type === 'itgov' ? 'it-governance' : 'trb';
 
   const isITGovAdmin = useContext(ITGovAdminContext);
-
   return (
     <div>
-      <PageHeading className="margin-y-0">{t('title')}</PageHeading>
-
+      <PageHeading className="margin-y-0">
+        {parentRoute === 'it-governance'
+          ? linkedSystemsT('title')
+          : adminT('title')}
+      </PageHeading>
       <p className="font-body-md line-height-body-4 text-light margin-top-05 margin-bottom-1">
-        {t('description')}
+        {parentRoute === 'it-governance'
+          ? linkedSystemsT('description')
+          : adminT('description')}
       </p>
-
       {(request.relationType === RequestRelationType.EXISTING_SYSTEM ||
         request.relationType === RequestRelationType.EXISTING_SERVICE) && (
         <div className="margin-bottom-3">
           <span className="font-body-md line-height-body-4 margin-right-1 text-base">
-            {t('somethingIncorrect')}
+            {adminT('somethingIncorrect')}
           </span>
 
           <UswdsReactLink
-            to={`/${parentRoute}/${request.id}/additional-information/link`}
+            to={`/${parentRoute}/${request.id}/system-information/link`}
           >
-            {t('editInformation')}
+            {parentRoute === 'it-governance'
+              ? linkedSystemsT('editInformation')
+              : adminT('editInformation')}
```

Comment:
This translation needs to be fixed.

<img width="445" height="184" alt="Screenshot 2025-08-01 at 1 45 12 PM" src="https://github.com/user-attachments/assets/1af01305-5c7a-4536-913a-5396c685fd68" />